modules/hello-world/examples/index-hello-world.js

```diff
@@ -0,0 +1,88 @@
+const couchbase = require('couchbase')
+
+async function go() {
+  const cluster = await couchbase.connect('couchbase://localhost', {
+    username: 'Administrator',
+    password: 'password',
+  })
+
+  console.log('[primary]')
+  // tag::primary[]
+  await cluster.queryIndexes().createPrimaryIndex(
+    'travel-sample',
+    // Don't error if the primary index already exists.
+    { ignoreIfExists: true }
+  )
+  // end::primary[]
```

Comment:
I think you can create and drop named primary indexes with the Node.js SDK: [CreatePrimaryQueryIndexOptions](https://docs.couchbase.com/sdk-api/couchbase-node-client/interfaces/CreatePrimaryQueryIndexOptions.html#name), [DropPrimaryQueryIndexOptions](https://docs.couchbase.com/sdk-api/couchbase-node-client/interfaces/DropPrimaryQueryIndexOptions.html#name). Not necessary, but please consider adding examples of this also.